app/config/config_dev.yml

```diff
@@ -14,9 +14,10 @@ web_profiler:
 monolog:
     handlers:
         main:
-            type:  stream
-            path:  "%kernel.logs_dir%/%kernel.environment%.log"
-            level: debug
+            type:     stream
+            path:     "%kernel.logs_dir%/%kernel.environment%.log"
+            level:    debug
```

Comment:
I suggest not to align the values, to avoid changing the indentation each time the config changes (btw, conf for other bundles is not aligning values)